csharp/test/Apache.Arrow.Tests/SchemaBuilderTests.cs

```diff
@@ -16,12 +16,41 @@
 using Apache.Arrow.Types;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using Xunit;
 
 namespace Apache.Arrow.Tests
 {
     public class SchemaBuilderTests
     {
+        public class SchemaComparer
```

Comment:
@pgovind - now that #3736 is merged, can you change the tests in that PR to call this new equality comparer?